Gemfile

```diff
@@ -44,7 +44,6 @@ group :development, :test do
   gem 'selenium-webdriver'
   gem 'rspec-rails', '>= 3.5.0'
   gem 'factory_bot_rails'
-  gem 'faker'
```

Comment:
Is this already on master? Is that why it's showing as a removal?

Reply:
Hmmm I had a bit of a nightmare that resulted in me uninstalling rails, bundle, ruby (dont ask!!! ha ha) - and when I reinstalled everything my gemfile was different... all tests were passing and it was just my gemfile so I did push it to master. Not sure if this is what you mean in this comment though because I also removed faker